crates/burn-cuda/src/lib.rs

```diff
@@ -17,6 +17,7 @@ mod tests {
     use burn_jit::JitBackend;
 
     pub type TestRuntime = cubecl::cuda::CudaRuntime;
+    pub use half::{bf16, f16};
 
-    burn_jit::testgen_all!();
+    burn_jit::testgen_all!(f32: [f16, bf16, f32], i32: [i8, i16, i32, i64]);
```

Comment:
Syntax question, why `f32: [f16, bf16, f32]` instead of `float: [f16, bf16, f32]` ? Do we assert the values in f32 that is why `f32` is used?

Reply:
`f32` is the "default" type, since I didn't want to do matrix testing (it would blow up test time to much). So we run all relevant tests with each float type and `i32` as the int type,  and then we run all the tests that use `Int` with  `f32` as the float type and then each int type. I think I can actually just use the type set on the default test backend though, so I think I'll remove this.

Reply:
It was mostly a carry over from `cubecl` where we don't have a backend that defines the default type